It is already not working

ERROR in ./node_modules/react-input-autosize/lib/AutosizeInput.js
Module not found: Error: Can't resolve 'create-react-class' in '/home/.../node_modules/react-input-autosize/lib'
 @ ./node_modules/react-input-autosize/lib/AutosizeInput.js 7:18-47
 @ ./node_modules/react-select-plus/lib/Select.js
...

--- a/bower.json
+++ b/bower.json
@@ -4,7 +4,7 @@
     "dist/react-select-plus.min.js",
     "dist/react-select-plus.min.css"
   ],
-  "version": "1.0.0-rc.3.patch4",
+  "version": "1.0.0-rc.3.patch5",
   "homepage": "https://github.com/HubSpot/react-select-plus",
   "authors": [
     "Trevor Burnham"

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-select-plus",
-  "version": "1.0.0-rc.3.patch4",
+  "version": "1.0.0-rc.3.patch5",
   "description": "A fork of react-select with support for option groups",
   "main": "lib/Select.js",
   "style": "dist/react-select-plus.min.css",
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.4",
-    "react-input-autosize": "^1.1.0"
+    "react-input-autosize": "1.1.0"
   },
   "devDependencies": {
     "babel": "^5.8.23",

